src/open_inwoner/accounts/tests/test_inbox_page.py

```diff
@@ -177,6 +198,46 @@ def test_no_contacts(self):
         # no form
         self.assertFalse(response.pyquery("form#message-form"))
 
+    def test_file_is_not_rendered_when_sharing_not_allowed(self):
+        config = SiteConfiguration.get_solo()
+        config.allow_messages_file_sharing = False
+        config.save()
+
+        response = self.app.get(self.url)
+        emoji = response.pyquery(".emoji")
+        file = response.pyquery(".message-file")
+
+        self.assertNotEqual(emoji, [])
+        self.assertEqual(file, [])
+
+    def test_file_field_doesnt_exist_when_sharing_not_allowed(self):
+        config = SiteConfiguration.get_solo()
+        config.allow_messages_file_sharing = False
+        config.save()
+
+        response = self.app.get(self.url)
+        form = response.forms["message-form"]
+
+        self.assertNotIn("file", form.fields.keys())
+
+    def test_file_cannot_be_uploaded_when_sharing_disabled(self):
+        config = SiteConfiguration.get_solo()
+        config.allow_messages_file_sharing = False
+        config.save()
+
+        contact = UserFactory()
+        self.user.user_contacts.add(contact)
+
+        image = Image.new("RGB", (10, 10))
+        byteIO = io.BytesIO()
+        image.save(byteIO, format="png")
+        img_bytes = byteIO.getvalue()
```

Comment:
Not very DRY: Similar to the feedback on the other PR (and some existing code): let's create one or more factories or function somewhere that generate image-files, byes and FileImage's.